### ISSUE TITLE
false positive no-member for static derived member

### ISSUE THREAD

**Issue by przecze** [PyCQA/pylint#3803]
I know there are problems for dynamically generated members, but in this case everything is static.
### Steps to reproduce
execute pylint on this code:
```
class Base:
    label: str

class Derived(Base):
    label = "I exist!"

print(Derived.label)
```
### Current behavior
```
test.py:7:6: E1101: Class 'Derived' has no 'label' member (no-member)
```
reported by pylint

### Expected behavior
No `no-member` error reported by pylint

### pylint --version output
```
~:$ pylint --version
pylint 2.6.0
astroid 2.4.2
Python 3.8.2 (default, Jul 16 2020, 14:00:26) 
[GCC 9.3.0]
```
tested also on
```
~:$ pylint --version
pylint 2.4.4
astroid 2.3.3
Python 3.8.2 (default, Jul 16 2020, 14:00:26) 
[GCC 9.3.0]
```

**Comment by hippo91** [comment]
@przecze thanks for the report.
I can reproduce it. An interesting fact is that the code below doesn't trigger the bug.
```
class Base:
    pass

class Derived(Base):
    label = "I exist!"

print(Derived.label)
```
